### DISCUSSION TITLE
Cant install while-labeled copy of android app side by side with official

### DISCUSSION THREAD

**Discussion by xklonx** [RocketChat/Rocket.Chat.ReactNative#3862] [Q&A] [answered]
I changed APPLICATION_ID, put my own google-services.json, then built master by gradlew bundleOfficialPlayRelease,
uploaded it to play store, downloaded apk, tried to install - application not installed.
Uninstalled official app - successfull install. Trying to install back official app - application not installed.
What im missing?

**Comment by diegolmello** [comment]
- Plug your device
- Run adb logcat
- Install the app and check the error

**Comment by xklonx** [comment]
Looks like i need to change android:sharedUserId in official/androidmanifest.xml

03-09 22:04:34.188  1677  2157 E PackageManager: Adding duplicate app id: 10445 name=chat.rocket.android
03-09 22:04:34.193  1677  2157 W PackageManager: Reconciliation failed...
03-09 22:04:34.193  1677  2157 W PackageManager: com.android.server.pm.PackageManagerService$ReconcileFailure: Reconcile failed: Package chat.rocket.android has no signatures that match those in shared user chat.rocket.android; ignoring!
03-09 22:04:34.193  1677  2157 W PackageManager: 	at com.android.server.pm.PackageManagerService.reconcilePackagesLocked(PackageManagerService.java:16942)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at com.android.server.pm.PackageManagerService.installPackagesLI(PackageManagerService.java:17334)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at com.android.server.pm.PackageManagerService.installPackagesTracedLI(PackageManagerService.java:16661)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at com.android.server.pm.PackageManagerService.lambda$processInstallRequestsAsync$22$PackageManagerService(PackageManagerService.java:14738)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at com.android.server.pm.-$$Lambda$PackageManagerService$9znobjOH7ab0F1jsW2oFdNipS-8.run(Unknown Source:6)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at android.os.Handler.handleCallback(Handler.java:938)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at android.os.Handler.dispatchMessage(Handler.java:99)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at android.os.Looper.loop(Looper.java:236)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at android.os.HandlerThread.run(HandlerThread.java:67)
03-09 22:04:34.193  1677  2157 W PackageManager: 	at com.android.server.ServiceThread.run(ServiceThread.java:45)
03-09 22:04:34.201  1677  2157 I InstallationStatistic: chat.rocket.android|com.android.vending|14746|14572|64|0|38|0
03-09 22:04:34.201  1374 16632 E installd: Couldn't opendir /data/app/vmdl1322235533.tmp: No such file or directory
03-09 22:04:34.201  1374 16632 E installd: Failed to delete /data/app/vmdl1322235533.tmp: No such file or directory
03-09 22:04:34.202  1677  2157 I com.android.server.pm.PackageManagerServiceInjector: MIUILOG-onPackageInstalled: PackageEvent { mTimeStamp=2022-03-09 22:04:34 mAction=Install mPackageName=chat.rocket.android mReturnCode=-8 mReturnMsg=Reconciliation failed...: Reconcile failed: Package chat.rocket.android has no signatures that match those in shared user chat.rocket.android; ignoring! mVersionCode=0 mVersionName=mInstallerPkgName=com.android.vending
03-09 22:04:34.204 11568 11568 E Finsky  : [2] ofn.onReceive(19): Error -505 while installing chat.rocket.android: INSTALL_FAILED_SHARED_USER_INCOMPATIBLE: Reconciliation failed...: Reconcile failed: Package chat.rocket.android has no signatures that match those in shared user chat.rocket.android; ignoring!
03-09 22:04:34.208 11568 11761 W Finsky  : [188308] ofr.F(5): Unexpected error abandoning session=1322235533. exception=java.lang.SecurityException: Caller has no access to session 1322235533
03-09 22:04:34.209 11568 11568 W Finsky  : [2] khw.e(4): Installer::IT: Install failure of chat.rocket.android (isid: IhwFzrxOTTSOP8ZeUsjCjA): -505, Exception: n/a, ErrorMessage: INSTALL_FAILED_SHARED_USER_INCOMPATIBLE: Reconciliation failed...: Reconcile failed: Package chat.rocket.android has no signatures that match those in shared user chat.rocket.android; ignoring!

**Comment by diegolmello** [comment, marked as answer]
`INSTALL_FAILED_SHARED_USER_INCOMPATIBLE` is the error you're looking for.
You did something wrong on your `android:sharedUserId` on manifests.
